app/controllers/pages_controller.rb

```diff
@@ -3,6 +3,9 @@ class PagesController < ApplicationController
   layout :layout_by_path
 
   def index
+    @nav = default_nav
+    puts @nav
```

Comment:
This can be removed.

Reply:
Thanks, good catch. I've pushed up the fix.